.version

```diff
@@ -1 +1 @@
-v1.7.1
\ No newline at end of file
+v1.7.2
```

Comment:
I'd like to see this move to 1.8.0 since we're adding a new feature.  (You'll need to change all the versions below as well.

Reply:
Changed.